tests/Feature/ActingAsTest.php

```diff
@@ -18,11 +20,18 @@ public function testActingAsWhenTheRouteIsProtectedByAuthMiddleware()
         /** @var Registrar $router */
         $router = $this->app->make(Registrar::class);
 
-        $router->get('/foo', function () {
+        $router->get('/foo', function (Request $request) {
+            $this->assertSame('client-1234', $request->user()->token()->oauth_client_id);
+            $this->assertSame(1234, $request->user()->token()->oauth_user_id);
+            $this->assertSame(['admin'], $request->user()->token()->oauth_scopes);
+
             return 'bar';
         })->middleware('auth:api');
 
-        Passport::actingAs(new User());
+        $client = new Client(['id' => 'client-1234']);
+        $user = (new User())->forceFill(['id' => 1234]);
```

Comment:
Force filling here is necessary, as the model is guarded. I tried to change as little as possible